File: server/src/main/java/gg/mineral/server/entity/object/Item.java
package gg.mineral.server.entity.object;

import gg.mineral.server.entity.EntityImpl;
import gg.mineral.server.world.WorldImpl;

public class Item extends EntityImpl {

    public Item(int id, WorldImpl world) {
        super(id, world);
    }

    @Override
    public void tickAsync() {
    }

}

File: server/src/main/kotlin/gg/mineral/server/entity/object/Item.kt
package gg.mineral.server.entity.`object`

import gg.mineral.server.entity.EntityImpl
import gg.mineral.server.snapshot.AsyncServerSnapshotImpl
import gg.mineral.server.world.WorldImpl

class Item(id: Int, serverSnapshot: AsyncServerSnapshotImpl, world: WorldImpl) : EntityImpl(id, serverSnapshot, world)
